content/smart.md

```diff
@@ -0,0 +1,140 @@
+- .NET: [https://github.com/ewoutkramer/fhir-net-api][FHIR-CLIENT-NET]
+- Python: [https://github.com/smart-on-fhir/client-py][FHIR-CLIENT-PYTHON]
+- iOS: [https://github.com/smart-on-fhir/Swift-FHIR][FHIR-CLIENT-IOS]
+
+### Tutorial ###
+
+Creating your first app can be overwhelming, so Cerner has created a [step by step tutorial][APP-TUTORIAL] on how to start creating your very first SMART app.  The app is written in HTML and JavaScript with some CSS.  We highly recommend that you take a look at the tutorial.
+
+### Code Console and Sandbox ###
+
+Cerner's implementation of SMART on FHIR ecosystem allows you to register, update, and delete your SMART app from [Code Console][CODE-CONSOLE].  Additionally, you can open and test your app from Code Console in our Sandbox environment.  Currently, there is a 15-minute waiting period for new or updated app configurations to take effect.  If you cannot open the application after the 15-minute period, post a question to the [Cerner FHIR Developers Google Group][CERNER-FHIR-DEV-GOOGLE-GROUP].
+
+### Launch Context ###
+
+See the [Scopes and Launch Content][SMART-HEALTH-IT-SCOPES-LAUNCH-CONTEXT] page for more information on how you can obtain the context. Generally, your app must able to retrieve SMART context parameters such as patient id, encounter id (if available), tenant, and a few others. You can inspect the OAuth2 access_token value that is returned to see the actual context.
+
+## Testing ##
+
+### Code Console ###
+
+For provider and patient facing apps, you can select any patient from a list of patients in our Sandbox environment to begin testing. After selecting a patient to test, you can open your app from a new web browser in [Code Console][CODE-CONSOLE]. Testing from Code Console through the browser helps you start developing and testing your app quickly.
+
+### PowerChart ###
+
+Once you have your app working in Code Console and you are committed to taking your app to production, contact us through the application available at [https://code.cerner.com/ehr-api][CODE-EHR-API], and we can provide access to *PowerChart* to test your app. For a provider facing apps, PowerChart is the environment where providers will access your app in a production environment, so it is very important to test the app in *PowerChart*.
+
+#### SSL Certificate ####
+
+Your app must be reachable through an https endpoint. Also, you need a valid SSL certificate for your site to test your app in *PowerChart*.
+
+#### Public Access ####
+
+In order for *PowerChart* to open and display your app, the URL of your app needs to be publicly accessible. In other word, the URL cannot point to hosts on your internal network.
+
+## Additional Resources ##
+
+- [Cerner Millennium FHIR API Documentation][CERNER-MILL-FHIR-API-DOC]
+- [Cerner Authorization Documentation][CERNER-AUTHZ-DOC]
+- [Cerner FHIR Developers Google Group][CERNER-FHIR-DEV-GOOGLE-GROUP]
+
+[HL7-FHIR]: https://www.hl7.org/fhir/
```

Comment:
Nit - I would much rather prefer you put these links inline where they are referenced. The only reason to put a named link at the bottom here is if we reference it a lot and just want the URL in one place. However, I don't think our links warrant this.

I know you were following how links are done on other pages here but I'm not sure why we started that. We shouldn't continue it here